### PR TITLE
Ls 4 added decorative animation effect, created mobile menu		

### PR DESCRIPTION
**Pull request by lyukhymchak**

### CODE REVIEW

File: html_css/css/style.css [hunk 1]
```diff
@@ -52,13 +52,24 @@ body {
...
   display: inline;
   text-decoration: none;
 }
-.logo:hover {
-  --color2: var(--black-color);
+.logo-svg {
+  width: 54px;
+  height: 24px;
+  transition: var(--color1) 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
```
Comment (philippbulanin) [inline html_css/css/style.css:65]: Please add blank lines beetwen all code blocks

File: html_css/css/style.css [hunk 2]
```diff
@@ -73,6 +84,57 @@ body {
...
+  align-items: flex-start;
+  flex-wrap: nowrap;
+  width: 100%;
+}
+
+.hamburger-menu .navbar__item {
+  padding-top: 32px;
+  padding-bottom: 32px;
+  width: 100%;
+  border-bottom: 1px solid #3d7fff20;
```
Comment (philippbulanin) [inline html_css/css/style.css:124]: Please use variarle for color

File: html_css/index.html [hunk 3]
```diff
@@ -229,9 +239,12 @@ <h3 class="courses__title">Learn Photoshop</h3>
                   </div>
                 </li>
               </ul>
-              <button class="btn-slider">
+              <svg class="right">
+                <use href="images/sprite.svg#icon-right"></use>
+              </svg>
+              <!-- <button class="btn-slider">
```
Comment (philippbulanin) [inline html_css/index.html:245]: Please remove commented code

File: html_css/index.html [hunk 4]
```diff
@@ -454,9 +469,12 @@ <h2 class="section-header title-center header-with-line">
               veritatis et quasi architecto beatae vitae dicta sunt explicabo.
             </p>
             <div class="preference-carousel">
-              <button class="btn-slider">
-                <div class="slider-left"></div>
-              </button>
+              <svg class="left">
+                <use href="images/sprite.svg#icon-left"></use>
+              </svg>
+              <!-- <button class="btn-slider">
```
Comment (philippbulanin) [inline html_css/index.html:475]: Please remove commented code

File: html_css/index.html [hunk 5]
```diff
@@ -471,9 +489,12 @@ <h2 class="section-header title-center header-with-line">
                   Education
                 </li>
               </ul>
-              <button class="btn-slider">
+              <svg class="right">
+                <use href="images/sprite.svg#icon-right"></use>
+              </svg>
+              <!-- <button class="btn-slider">
```
Comment (philippbulanin) [inline html_css/index.html:495]: Please remove commented code

File: html_css/js/mobile-menu.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+let hamburger = document.getElementsByClassName("hamburger-menu-logo")[0];
```
Comment (philippbulanin) [inline html_css/js/mobile-menu.js:1]: Use const for static variables

File: html_css/js/mobile-menu.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+console.log(hamburger);
```
Comment (philippbulanin) [inline html_css/js/mobile-menu.js:2]: Remove console log

File: html_css/js/mobile-menu.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+hamburger.onclick = function (){
+    let navbar = document.getElementsByClassName("navbar")[0];
```
Comment (philippbulanin) [inline html_css/js/mobile-menu.js:4]: Use const for static variables

File: html_css/js/mobile-menu.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+    console.log(navbar);
```
Comment (philippbulanin) [inline html_css/js/mobile-menu.js:5]: Remove console log

File: html_css/js/mobile-menu.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+    navbar.classList.add("hamburger-menu");
+    let closeBtn = document.getElementsByClassName("close-svg")[0];
```
Comment (philippbulanin) [inline html_css/js/mobile-menu.js:7]: Use const for static variables

File: html_css/js/mobile-menu.js [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+    closeBtn.onclick = function () {
```
Comment (philippbulanin) [inline html_css/js/mobile-menu.js:8]: Please mention, that in this case you creating new listener on every menu opening. You should avoid it

**Review by philippbulanin — CHANGES_REQUESTED** [review]

**Comment by lyukhymchak** [comment]
@philippbulanin 
Issues have been fixed.

**Pull request merged**